easybuild/easyconfigs/h/HPCX/HPCX-2.3.0-GCC-8.2.0-2.31.1.eb

```diff
@@ -0,0 +1,18 @@
+easyblock = 'Bundle'
+
+name = 'HPCX'
+version = '2.3.0'
+
+homepage = 'http://www.mellanox.com/page/products_dyn?product_family=189&mtag=hpc-x'
+description = """The Mellanox HPC-X Toolkit
+is a comprehensive MPI and SHMEM/PGAS software suite
+for high performance computing environments"""
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
+
+dependencies = [
+    ('UCX', '1.5.0rc1', '-hpcx', ('GCCcore', '8.2.0')),
+    ('OpenMPI', '4.0.x', '-hpcx', ('GCC', '8.2.0-2.31.1'))
```

Comment:
You're still requesting 4.0.x version, but the OpenMPI easyconfig below is 4.0.0. You have to decide if you want a 4.0.0 version or the "4.0.x" version. Do not call the OpenMPI easyconfig 4.0.0 if it is not using the official 4.0.0 release.

Reply:
Argh, missed that. Thanks! Version renamed to `4.0.x`. Is OK?